### PR TITLE
updating the readme file

### PR DESCRIPTION
**Pull request by janet6868**

### CODE REVIEW

File: data_preparation/crop_statistics_ML/README.md [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
+ICRISAT
+
+## Data card author
+Janet Mutuku, Celeste Tchampi, Pierre C. Sibiry Traore
+
+## Dataset overview
+**Crops**: Maize
+
+**Variables** [unit]:'country_code', 'cmdt_zone', 'cmdt_sector', 'adm_id', 'season_name',
+       'harvest_year', 'crop_name', 'production', 'planted_area', 'yield'
```
Comment (krsnapaudel) [inline data_preparation/crop_statistics_ML/README.md:23]: These are variables in the original data, not in the processed data. This should go back to the previous one.

File: data_preparation/crop_statistics_ML/shapefiles_ML/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Compagnie Malienne pour le Developpement des Textiles (CMDT) Monitoring Operational Maize Yield Statistics
+
+## Short description
+The data derived comes from CMDT Monitoring Operational - a Malian Company for the Development of Textiles located in Southern Mali. It consists of 5 CMDT regions and 28 CMDT sectors. It provides key observations from the maize yield data for the different zones/regions and sectors from 1990 to 2017. There is significant spatial and temporal variation in maize productivity in the region, likely driven by differences in agro-climatic conditions, management practices, and potential data limitations. 
+
+## Link
+
```
Comment (krsnapaudel) [inline data_preparation/crop_statistics_ML/shapefiles_ML/README.md:7]: Could say "Prepared for AgML-CY-Bench."

**Review by krsnapaudel — CHANGES_REQUESTED** [review]

File: data_preparation/crop_statistics_ML/shapefiles_ML/README.md (hunk 2 above)
Comment (janet6868) [reply data_preparation/crop_statistics_ML/shapefiles_ML/README.md:7]: yes

File: data_preparation/crop_statistics_ML/README.md (hunk 1 above)
Comment (janet6868) [reply data_preparation/crop_statistics_ML/README.md:23]: These variables have been excluded. The readme in the shapefile contains these: 'country_code', 'cmdt_zone', 'cmdt_sector', 'adm_id'

File: data_preparation/crop_statistics_ML/shapefiles_ML/README.md (hunk 2 above)
Comment (krsnapaudel) [reply data_preparation/crop_statistics_ML/shapefiles_ML/README.md:6]: I meant in the Link section.

**Review by krsnapaudel — CHANGES_REQUESTED** [review]
You need the following organization:
```
data_preparation/crop_statistics_ML
data_preparation/shapefiles_ML
```
So move `shapefiles_ML` to under `data_preparation`.

**Comment by janet6868** [comment]
@krsnapaudel, updated.
Thanks

**Review by krsnapaudel — APPROVED** [review]

**Pull request merged**